### DISCUSSION TITLE
high priority order on the same agv

### DISCUSSION THREAD

**Discussion by akashjinandra** [usnistgov/ARIAC#225] [Q&A]
Hello, is it possible that a high priority order will be on the same agv as one of the lower priority orders that are still running? For instance you just placed a part on agv2, can the high priority order also be on agv2? in which case the user would have to take off the old parts and put new ones on?

**Comment by jaybrecht** [comment]
I think this was possibly answered elsewhere but no for ARIAC2023 if a agv is used for a kitting order there will not be another order that can be on that AGV
